example/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/kubernetes-csi/csi-lib-iscsi/iscsi"
+)
+
+var (
+	portals   = flag.String("portals", "192.168.1.107:3260", "Comma delimited.  Eg: 1.1.1.1,2.2.2.2")
+	iqn       = flag.String("iqn", "iqn.2010-10.org.openstack:volume-eb393993-73d0-4e39-9ef4-b5841e244ced", "")
+	multipath = flag.Bool("multipath", false, "")
+	username  = flag.String("username", "86Jx6hXYqDYpKamtgx4d", "")
+	password  = flag.String("password", "Qj3MuzmHu8cJBpkv", "")
+	lun       = flag.Int("lun", 1, "")
+)
+
+func main() {
+	flag.Parse()
+	tgtp := strings.Split(*portals, ",")
+
+	c := iscsi.Connector{
+		AuthType:      "chap",
+		TargetIqn:     *iqn,
+		TargetPortals: tgtp,
+		SessionSecrets: iscsi.Secrets{
+			UserName: "",
+			Password: ""},
+		Lun:           int32(*lun),
+		Multipath:     *multipath,
```

Comment:
@j-griffith It seems that its possible to configure multipath here from the connector, however failed to interpret how passive or backup portals can be mentioned ? May be via targetportals slice, however I dont see any extra attempts to login via this portals in the code. Can you shed some light on this ?

Reply:
Hi @humblec this is just your standard linux multipath driver here, it's not attempting to do anything more sophisticated like stand-by paths etc.  For usage here, I expect multipath tools to be installed on the node; from there the mp drivers pretty much handle multiple routes dynamically and it's left to that.  I should call that out in the docs.